compiler/backend/backends.nim

```diff
@@ -879,8 +880,10 @@ iterator discover*(env: var MirEnv, progress: EnvCheckpoint
   ## processing -- don't use it in conjunction with the ``process`` iterator.
   for id, it in since(env.constants, progress.consts):
     # first discover and report the procedures referenced by the constant's
-    # data, allowing
+    # data. This ensures that the callsite can rely on all the constant's
+    # dependencies existing in the environment
```

Comment:
There's no change in behaviour here, I just forgot to finish the sentence in the original PR.